src/invocation.js

```diff
@@ -0,0 +1,80 @@
+import { Address } from './address';
+import { scValToNative } from './scval';
+
+/**
+ * @typedef CreateInvocation
```

Comment:
are these actual types somewhere?

Reply:
this is just an api proposal, so I was trying to use jsdocs to define the types